app/src/main/java/com/mashup/data/repository/DanggnRankRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.mashup.data.repository
+
+import com.mashup.data.dto.DanggnAllMemberRankResponse
+import com.mashup.data.dto.DanggnMemberRankResponse
+import com.mashup.data.dto.DanggnPlatformRankResponse
+import com.mashup.network.Response
+import com.mashup.network.dao.DanggnRankDao
+import javax.inject.Inject
+
+class DanggnRankRepository @Inject constructor(
+    private val danggnRankDao: DanggnRankDao
+) {
+    suspend fun getPersonalDanggnRank(
+        generationNumber: Int,
+        limit: Int
+    ): Response<DanggnMemberRankResponse> {
```

Comment:
```suggestion
    ): Response<List<MemeberRankResponse>> {
```

그래서 이렇게 정의하면 될 것 같네요!!

아래도 부탁드립니다 🤺

Reply:
[33c611d](https://github.com/mash-up-kr/mashup_Android/pull/272/commits/33c611d303a4fb0d81e47d1cd8bf0b8d2957a73b) 수정 완료